pkg/crc/services/dns/dns_linux.go

```diff
@@ -8,10 +8,7 @@ import (
 func runPostStartForOS(serviceConfig services.ServicePostStartConfig, result *services.ServicePostStartResult) (services.ServicePostStartResult, error) {
 	// We might need to set the firewall here to forward
 	// Update /etc/hosts file for host
-	if err := goodhosts.UpdateHostsFile(serviceConfig.IP, serviceConfig.BundleMetadata.GetAPIHostname(),
-		serviceConfig.BundleMetadata.GetAppHostname("oauth-openshift"),
-		serviceConfig.BundleMetadata.GetAppHostname("console-openshift-console"),
-		serviceConfig.BundleMetadata.GetAppHostname("default-route-openshift-image-registry")); err != nil {
+	if err := goodhosts.UpdateHostsFile(serviceConfig.IP, serviceConfig.BundleMetadata.GetAPIHostname()); err != nil {
```

Comment:
how does this work on cluster startup? do we get a sync for all the routes the cluster knows? or would we need to add it ourself? I guess on cluster stop it would also be nice to remove all the known routes from /etc/hosts?